components/menu/stories/template.js

```diff
@@ -258,6 +274,7 @@ export const MenuItem = (
 		hasExternalLink = false,
 		hasActions = false,
 		id = getRandomId("menuitem"),
+		labelId = getRandomId("menuitem-label"),
```

Comment:
label id lets us connect the switch input field with the already present label for the menu item